client/my-sites/email/mailboxes/constants.ts

```diff
@@ -1,6 +1,4 @@
 /**
  * The value `inbox` often used as a `source` url parameter
- *
- * @type {string}
  */
-export const INBOX_SOURCE = 'inbox';
+export const MAILBOXES_SOURCE = 'mailboxes';
```

Comment:
```suggestion
export const INBOX_SOURCE = 'inbox';
```
I'm inclined to leave this as-is to allow data continuity. By changing this now, we'll have disconnected data where traffic prior to deploying this change will be reported against `inbox` and traffic after deploying this change will be reported against `mailboxes`.

Reply:
Fine with me. I'll revert.